setup.cfg

```diff
@@ -60,3 +60,14 @@ docs =
 
 [options.package_data]
 * = data/*.csv, data/cmaps/*.txt
+
+[tool:pytest]
+markers =
+    advanced: marks tests as advanced (deselect with '-m "not advanced"')
+filterwarnings =
+    ignore::DeprecationWarning:pkg_resources
```

Comment:
Will these cause issues in the future if they're all hidden?

Reply:
Probably not, since this is package specific and it's an issue of the package itself needing to be updated.